AxonDeepSeg/morphometrics/compute_morphometrics.py

```diff
@@ -324,11 +335,12 @@ def save_map_of_axon_diameters(path_folder, axon_diameter_figure):
     axon_diameter_figure.savefig(file_path)
 
 
-def get_aggregate_morphometrics(pred_axon, pred_myelin, path_folder):
+def get_aggregate_morphometrics(pred_axon, pred_myelin, path_folder, circle_approx = True):
```

Comment:
fix PEP8-- no space around "="-- and do it everywhere in your changes

Reply:
Resolved in 3ba42bc